jenkins/jenkinsfile

```diff
@@ -0,0 +1,32 @@
+pipeline {
+    agent {
+         docker {
+            label 'AL2-X64'
+            /* See  
+            https://github.com/opensearch-project/opensearch-build/blob/main/docker/ci/dockerfiles/build.ubuntu18.opensearch.x64.dockerfile
```

Comment:
Is this still the right URL for the `2.0` backport? I only ask because I see `/main/` in there

Reply:
We can merge this if it looks fine and then I can backport https://github.com/opensearch-project/OpenSearch/pull/2656 to 2.0.